embedded/store/immustore.go

```diff
@@ -2571,13 +2571,17 @@ func (s *ImmuStore) readValueAt(b []byte, off int64, hvalue [sha256.Size]byte, s
 		val, err := s.vLogCache.Get(off)
 		if err == nil {
 			// the requested value was found in the value cache
-			copy(b, val.([]byte))
+			bval := val.([]byte)
+			n := len(bval)
```

Comment:
we may not want to redeclare this variable as it is already defined in the return parameter